src/display/annotation_layer.js

```diff
@@ -530,6 +532,66 @@ var TextWidgetAnnotationElement = (
 
 /**
  * @class
+ * @alias ChoiceWidgetAnnotationElement
+ */
+var ChoiceWidgetAnnotationElement = (
+    function ChoiceWidgetAnnotationElementClosure() {
+  function ChoiceWidgetAnnotationElement(parameters) {
+    WidgetAnnotationElement.call(this, parameters);
```

Comment:
When we only supported `TextWidgetAnnotation`s, I suppose it didn't matter too much that we set the `isRenderable` parameter inside of `WidgetAnnotationElement`, however now I don't think that's correct anymore!
*Note:* For `TextWidgetAnnotation` there's a meaningful fallback in the case where no `Appearance` stream existed, and was thus not rendered in the `core` layer, but for the `ChoiceWidgetAnnotationElement` we're not rendering anything in the `display` layer by default.

I believe that we should to modify the current code like below, which will help not only here but when implementing other widget's later.
```diff
diff --git a/src/display/annotation_layer.js b/src/display/annotation_layer.js
index 29ed79a..76a8b73 100644
--- a/src/display/annotation_layer.js
+++ b/src/display/annotation_layer.js
@@ -400,9 +400,7 @@ var TextAnnotationElement = (function TextAnnotationElementClosure() {
  * @alias WidgetAnnotationElement
  */
 var WidgetAnnotationElement = (function WidgetAnnotationElementClosure() {
-  function WidgetAnnotationElement(parameters) {
-    var isRenderable = parameters.renderInteractiveForms ||
-      (!parameters.data.hasAppearance && !!parameters.data.fieldValue);
+  function WidgetAnnotationElement(parameters, isRenderable) {
     AnnotationElement.call(this, parameters, isRenderable);
   }
 
@@ -432,7 +430,9 @@ var TextWidgetAnnotationElement = (
   var TEXT_ALIGNMENT = ['left', 'center', 'right'];
 
   function TextWidgetAnnotationElement(parameters) {
-    WidgetAnnotationElement.call(this, parameters);
+    var isRenderable = parameters.renderInteractiveForms ||
+      (!parameters.data.hasAppearance && !!parameters.data.fieldValue);
+    WidgetAnnotationElement.call(this, parameters, isRenderable);
   }
 
   Util.inherit(TextWidgetAnnotationElement, WidgetAnnotationElement, {
```

Then this line becomes
```javascript
WidgetAnnotationElement.call(this, parameters, parameters.renderInteractiveForms);
```
and the `if (this.renderInteractiveForms) {` condition is no longer necessary inside of `render`.